### ISSUE TITLE
After connecting multiples nodes shoukaku player not give sounds to other voice regions

### ISSUE THREAD

**Issue by vkamsingh** [Deivu/Shoukaku#104]
why shoukaku search same song in every node while multiple nodes are connected? also after this other voice regions not get sound

**Comment by pasindu20** [comment]
> why shoukaku search same song in every node while multiple nodes are connected? also after this other vc regions not get sound

yes i also had this issue

**Comment by pasindu20** [comment]
look at closed issue #101 it had this same problem

**Comment by pasindu20** [comment]
> look at closed issue #101 it had this same problem

issue was not solved yet.

**Comment by vkamsingh** [comment]
@pasindu20 I think issue is not from our side it happend from shoukaku

**Comment by pasindu20** [comment]
Also i use shoukaku ripped off package call kazagumo https://www.npmjs.com/package/kazagumo idk this problem coming that package or shkoukaku

**Comment by Deivu** [comment]
First of all, I run 2 nodes on my bot without any issues

Screenshot one:
https://safe.kashima.moe/tbp0oigvnuef.png

Second of all voice region has no issues
Video about voice regions:
https://safe.kashima.moe/fi733176wani.mp4

So how can I fix something that isn't broken in the first place? 

Also Kazagumo is not a ripoff, thats a queue for Shoukaku to simplify the use of this library as I dont provide any queue for my library and leave it to developers to make one themselves

**Comment by Deivu** [comment]
Also Shoukaku just fowards voice packets to Lavalink, Lavalink does the reconnecting when voice region or channels are changed

**Comment by Deivu** [comment]
Closing due to inactivity

**Comment by pasindu20** [comment]
Bruh issue was not fixed

**Comment by Deivu** [comment]
I can't replicate your issue, and you didn't even tell me how are you getting this issue, if I can't replicate it. how can I fix it?

**Comment by Deivu** [comment]
Then I also don't know how to fix it. You can't even tell me how this issue happens. I connect 2 nodes myself without any issues, Other bots connect 10 nodes without any issues. Saying it happens after connecting 2 lava nodes doesn't say anything

**Comment by pasindu20** [comment]
After that some guilds not give voice for some regions like it happend randomly for guilds

**Comment by pasindu20** [comment]
I also don't know why is that happening

**Comment by pasindu20** [comment]
With 1 node it working fine

**Comment by Deivu** [comment]
At this point, I'm pretty sure it's a your code issue. Shoukaku has nothing to do with the actual voice being sent to tour voice channel. If you are well versed with how discord voice works, you would know what I mean. This library just forwards voice packets to Lavalink (server to connect to), and that's all I need to do. I establish a voice connection through your Discord websocket, receives it, then forwards it to Lavalink then Lavalink is the one responsible keeping the connection. Shoukaku sends every voice state and voice server update to Lavalink

**Comment by pasindu20** [comment]
I check lavalink logs with 2 node (if bot search some thing it will search both of lavaserver not only 1 idk that the issue here if voice region change both lavalink get updated )

**Comment by Deivu** [comment]
I suggest learning how things work, specially in your case. I'm sure you are doing a dark magic around your code. I never had that issue nor encountered that

**Comment by pasindu20** [comment]
> why shoukaku search same song in every node while multiple nodes are connected? also after this other voice regions not get sound

This guy happend that too

**Comment by Deivu** [comment]
And no matter what you tell me, I can't reproduce any of your issue in this thread. I tried reproducing your issue even you can't tell me anything clear yet I can't reproduce it. You are doing something very wrong, or you are misusing this library that's all

**Comment by Deivu** [comment]
TLDR: I can't reproduce it = Not my issue = Your code issue

Things would be different if I do reproduce it, but since I can't, this is **not** an issue with this lib then
